.env.example

```diff
@@ -15,3 +15,4 @@ VUE_APP_BATCH_JOB_ENUMS={"JOB_BKR_ORD_UNF":{"id":"JOB_BKR_ORD_UNF","facilityId":
 VUE_APP_WEBHOOK_ENUMS={"NEW_PRODUCTS":"products/create","DELETE_PRODUCTS":"products/update","NEW_ORDERS":"orders/create","CANCELLED_ORDERS":"orders/cancelled","PAYMENT_STATUS":"orders/paid","RETURNS":"","BULK_OPERATIONS_FINISH":"bulk_operations/finish"}
 VUE_APP_PERMISSION_ID=
 VUE_APP_ALIAS=
+VUE_APP_JOB_ENUM_TYPE={"ORDER_SYS_JOB": "VUE_APP_ODR_JOB_ENUMS", "PRE_ORD_SYS_JOB": "VUE_APP_PRODR_JOB_ENUMS", "PRODUCT_SYS_JOB": "VUE_APP_PRD_JOB_ENUMS", "INVENTORY_SYS_JOB": "VUE_APP_INV_JOB_ENUMS"}
```

Comment:
We don't need an extra entry here for enum maps, we can use spread syntax to have all the enums in a single variable. Take reference from JobDetails page.